packages/tss-client/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@toruslabs/tss-client",
-  "version": "3.0.0",
+  "version": "3.1.0",
   "description": "Client SDK for MPECDSA signing",
   "files": [
     "dist"
   ],
-  "main": "dist/tssClient.cjs.js",
-  "module": "dist/tssClient.esm.js",
+  "main": "dist/lib.cjs/index.js",
+  "module": "dist/lib.esm/index.js",
```

Comment:
for my understanding, why this change? the previously referenced files still exist.

Reply:
its some build optimization chai has done in torus-scripts, new files are more tree shakeable. @chaitanyapotti can share more about those optimizations.